apps/api/src/app/workflows-v2/shared/schemas/workflow-zod-schemas.ts

```diff
@@ -0,0 +1,63 @@
+import { z } from 'zod';
```

Comment:
If I am not mistaken, these DTOs are requests and responses for workflow and step endpoints. I would stick to the strategy we already have in the repo to define them so that, if I am not mistaken, it's not using Zod.

Although the DX that zod offers is much better, consistency and reusability is key in order for us to move fast. So this work should be part of a separate refactoring project we will do in the future..

Reply:
I'll remove it, it was there to help with the e2e tests, unsuccessfully